background.ts

```diff
@@ -63,6 +63,8 @@ Electron.app.whenReady().then(async() => {
   } catch (err) {
     console.log(`Can't get app version: ${ err }`);
   }
+  const isFirstRun = await settings.isFirstRun();
```

Comment:
Because `settings.isFirstRun()` was now being invoked after the `settings.json` file would be created. I verified this, but can't tell you which change(s) caused that.

Reply:
That should be okay, because `isFirstRun` just reads off a variable that was set in `init()`. But this works too.

Reply:
The `isFirstRun` calc needs a second look - I don't think it's actually working in main. Back into draft mode…